Source/Widgets/CesiumInspector/CesiumInspectorViewModel.js

```diff
@@ -104,247 +104,243 @@ define([
          * @type {Boolean}
          * @default false
          */
-        this.frustums = false;
+        this.frustums = knockout.observable(false);
```

Comment:
We use the knockout es5 plugin instead of creating observables directly so users can access observable properties the same way they would regular ones.  So instead of creating the observable here, `knockout.track` creates all the observables behind the scenes.